Extension/src/LanguageServer/configurations.ts

```diff
@@ -441,7 +441,10 @@ export class CppProperties {
         // Valid compiler + IntelliSenseMode combinations:
         // 1. compiler is cl/clang-cl and IntelliSenseMode is MSVC
         // 2. compiler is not cl/clang-cl and IntelliSenseMode is not MSVC
-        const isValid: boolean = compilerPathAndArgs.compilerName.endsWith("cl.exe") === configuration.intelliSenseMode.startsWith("msvc");
+        const isValid: boolean = compilerPathAndArgs.compilerName.endsWith("cl.exe") ===
```

Comment:
Not your change, but could you also update this to do a case-insensitive check against "cl.exe"?  i.e. Right now, if "CL.exe" is specified, it works, but we show a squiggle.